.github/pull_request_template.md

```diff
@@ -14,8 +14,10 @@ Include a summary of the changes and a link to the related issue. List any depen
 
 ## Checklist before requesting review
 - [ ] I have performed a self-review of my code
-- [ ] I have run the code that is being changed under ideal conditions, and it doesn't fail
```

Comment:
This one has been a helpful reminder for me.  Otherwise, I'm totally happy with these changes.  Thanks!